client/app/adapters/node.js

```diff
@@ -16,7 +16,7 @@ function getToken() {
 
 export default DS.JSONAPIAdapter.extend({
     session: Ember.inject.service(),
-    host: 'https://staging-api.osf.io',
+    host: 'https://api.osf.io',
```

Comment:
Can you change the host back to staging? It should really be a settings variable, but that change can go in separately since it's unrelated to the submission form.